org.nmap.Zenmap.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Kevin Degeling -->
+
+<component type="desktop-application">
+  <id>org.nmap.Zenmap</id>
+  <name>Zenmap</name>
+  <summary>Zenmap is the official Nmap Security Scanner GUI.</summary>
```

Comment:
Ditto.

Reply:
Also, a shorter summary like "The official Nmap Security Scanner GUI", or the same text as in the desktop file Comment ("A cross-platform GUI for the Nmap Security Scanner") would be better. ;-)